Proposals/xGov-21-Reduce unemployment with NFT Skills Certification.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Reduce unemployment with NFT Skills Certification
+author: Diki karim @dikikarim2010, Diki karim dikikarim2010@gmail.com
+company_name: none
+category: dApps
+focus_area: Education,NFT,Social
+open_source: Yes
+amount_requested: 1000000
```

Comment:
If your project requires multiple milestones, split them into multiple proposals. See: https://github.com/algorandfoundation/ARCs/pull/209

Reply:
> If your project requires multiple milestones, split them into multiple proposals. See: [algorandfoundation/ARCs#209](https://github.com/algorandfoundation/ARCs/pull/209)

thank you for your information